### PR TITLE
Allow models to specify variables to ignore

### PR DESCRIPTION
**Pull request by gabrieltseng**

### CODE REVIEW

File: tests/models/test_data.py [hunk 1]
```diff
@@ -113,27 +114,25 @@ def __init__(self):
...
         if experiment == 'nowcast':
-            expected_shape = (25, 3) if surrounding_pixels is None else (9, 3 * 9)
-            assert x_train_data.current.shape == expected_shape, "" \
-                "Expecting multiple vars" \
-                "in the current timestep. Expect: (25, 3) "\
-                f"Got: {x_train_data.current.shape}"
+            expected_shape = (25, 2) if surrounding_pixels is None else (9, 2 * 9)
+            assert x_train_data.current.shape == expected_shape, \
+                f'Expecting multiple vars in the current timestep. ' \
+                f'Expect: (25, 3) Got: {x_train_data.current.shape}'
```
Comment (tommylees112) [inline tests/models/test_data.py]: `Expect: (25, 2)` in the string

File: src/models/data.py [hunk 2]
```diff
@@ -201,6 +206,10 @@ def ds_folder_to_np(self,
         x, y = xr.open_dataset(folder / 'x.nc'), xr.open_dataset(folder / 'y.nc')
         assert len(list(y.data_vars)) == 1, f'Expect only 1 target variable! ' \
             f'Got {len(list(y.data_vars))}'
+
+        if self.ignore_vars is not None:
+            x = x.drop(self.ignore_vars)
```
[the remark this answers is not in the document]
Comment (gabrieltseng) [reply src/models/data.py]: I figured the xarray statement would be enough?

File: tests/models/test_data.py (hunk 1 above)
Comment (gabrieltseng) [reply tests/models/test_data.py]: Thanks! fixed

**Comment by gabrieltseng** [comment]
Thanks Tommy :)

**Pull request merged**